components/Banner/examples/supplemental/MDCBannerView.m

```diff
@@ -18,7 +18,7 @@
 #import "MaterialTypography.h"
 
 typedef NS_ENUM(NSInteger, MDCBannerViewLayoutStyle) {
-  MDCBannerViewLayoutSingleLineStyle = 0,              // All elements lays on the same line
+  MDCBannerViewLayoutSingleLineStyle = 0,              // All elements on the same line
```

Comment:
Please limit this PR to the examples. This file looks like it should be moved to a different PR.

Reply:
Sorry. I made some code review change, and cherry-picked to both PR. The diff didn't display it correctly. I just rebased this branch, and the diff should now show correctly.